docs/content/users/quickstart.md

```diff
@@ -211,6 +211,70 @@ ddev launch
 
 Visit [Ibexa documentation](https://doc.ibexa.co/en/latest/getting_started/install_with_ddev/) for more cases.
 
+## Kirby CMS
+
+Start a new [Kirby CMS](https://getkirby.com) project or use an existing one.
+
+=== "New projects"
```

Comment:
I am not sure if it would make sense for consistency reasons to change `New projects` to `Composer` and `Existing projects` to `Git Clone`, that way the tabs would be consistent with other CMS quickstart guides. Cuz the existing example is simply using git while the new project is using composer.

Reply:
ah yep you are right. but from my point of view the two labels for the tabs are ok. i just tried to make the tab labels consistent across quickstarts. but that is probably out of the scope of this PR since there are more quickstarts with the new and existing project pattern. probably it makes sense to make the quickstarts consistent in a single followup issue after https://github.com/ddev/ddev/pull/5058 went in.